extensions/wikia/JSVariables/JSVariables.php

```diff
@@ -78,7 +78,12 @@ function wfJSVariablesTopScripts(Array &$vars, &$scripts) {
 
 	$vars['wgTransactionContext'] = Transaction::getAttributes();
 
-	$scripts .= Html::inlineScript("var wgNow = new Date();") .	"\n";
+	$scripts .= Html::inlineScript("var wgNow = new Date();") . "\n";
+
+	// ADEN-6676: Inline InstantGlobals to prevent AdBlock block
```

Comment:
It's an open source. I'd suggest removing we do that ;) Just in case. Ticket number should be enough

Reply:
Right, removed.